libraries/botbuilder/tests/teams/composeMessagingExtensionAuth/src/composeMessagingExtensionAuthBot.ts

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+    Attachment,
+    CardFactory,
+    MessagingExtensionActionResponse,
+    MessagingExtensionAction,
+    MessagingExtensionQuery,
+    TaskModuleContinueResponse,
+    TaskModuleTaskInfo,
+    TaskModuleRequest,
+    TeamsActivityHandler,
+    TurnContext,
+} from 'botbuilder';
+
+import {
+    IUserTokenProvider,
+} from 'botbuilder-core';
+
```

Comment:
Can we put some notes here on how to invoke the bot ala the comment on C# side?

Reply:
added comment